tests/lax_numpy_indexing_test.py

```diff
@@ -871,6 +871,15 @@ def testTrivialGatherIsntGenerated(self):
     jaxpr = jax.make_jaxpr(lambda x: x[:4])(np.arange(4))
     self.assertEqual(len(jaxpr.jaxpr.eqns), 0)
 
+  @parameterized.named_parameters(
+      {"testcase_name": f"_{idx}", "idx": idx}
+      for idx in (-3, 0, 5, np.array(6), jnp.array(7)))
```

Comment:
Creating a `jnp.array` here at the global scope is not supported in some of our test frameworks, because the backend has not yet been initialized. I'd suggest doing something like
```
for idx in (-3, 0, 5)
for idx_type in (int, np.array, jnp.ndarray)
```
and then convert idx as necessary.

Reply:
:+1: changed it.